crates/btcio/src/broadcaster/task.rs

```diff
@@ -35,147 +35,144 @@ pub async fn broadcaster_task(
             Some((idx, txentry)) = entry_receiver.recv() => {
                 let txid: Option<Txid> = ops.get_txid_async(idx).await?.map(Into::into);
                 info!(%idx, ?txid, "Received txentry");
```

Comment:
```suggestion
                info!(%idx, %txid, "Received txentry");
```
We don't need `Debug` for `Txid` it has a nice `Display` impl.